src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -664,7 +664,12 @@ private void initShowTrackingNotification() {
 
                 @Override
                 public void run() {
-                    DefaultTaskExecutor.runInJavaFXThread(JabRefFrame.this::showTrackingNotification);
+                    SwingUtilities.invokeLater(new Runnable() {
```

Comment:
It would be nice if you could convert this to a lambda expression. (Unfortunately this lamdba styles is not possible by javafx)
Just:
```
    SwingUtilities.invokeLater(() -> {
                        DefaultTaskExecutor.runInJavaFXThread(JabRefFrame.this::showTrackingNotification);
                    });
```

Reply:
Yes, this definitely looks cleaner. Applied and re-tested.